src/DssSpell.t.sol

```diff
@@ -65,29 +65,27 @@ contract DssSpellTest is DSTest, DSMath {
     Rates rates;
 
     // MAINNET ADDRESSES
-    DSPauseAbstract      pause = DSPauseAbstract(    0xbE286431454714F511008713973d3B053A2d38f3);
-    address         pauseProxy =                     0xBE8E3e3618f7474F8cB1d074A26afFef007E98FB;
-    DSChiefAbstract      chief = DSChiefAbstract(    0x9eF05f7F6deB616fd37aC3c959a2dDD25A54E4F5);
-    VatAbstract            vat = VatAbstract(        0x35D1b3F3D7966A1DFe207aa4514C12a259A0492B);
-    VowAbstract            vow = VowAbstract(        0xA950524441892A31ebddF91d3cEEFa04Bf454466);
-    CatAbstract            cat = CatAbstract(        0xa5679C04fc3d9d8b0AaB1F0ab83555b301cA70Ea);
-    PotAbstract            pot = PotAbstract(        0x197E90f9FAD81970bA7976f33CbD77088E5D7cf7);
-    JugAbstract            jug = JugAbstract(        0x19c0976f590D67707E62397C87829d896Dc0f1F1);
-    SpotAbstract          spot = SpotAbstract(       0x65C79fcB50Ca1594B025960e539eD7A9a6D434A3);
+    ChainlogAbstract constant CHANGELOG = ChainlogAbstract(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
```

Comment:
I think I agree with this.  While it's cleaner to use the `ChainLog` in the tests, we might want the test to challenge our assumptions and configuration by explicitly testing addresses.

On the other hand, making spell construction easier is a huge part of why we have the `ChainLog`.  It seems a shame not to lean on it here.

Perhaps a hybrid approach that gets us easy test construction, but also has a relatively static section that validates all `ChainLog` addresses?  The same way we enumerate system and collateral config, we would have a test that makes sure `ChainLog` is correct.

Reply:
I actually do agree with @kmbarry1 after thinking about this some more. One thing that we can do to make it easier on ourselves is to avoid removing addresses from PR to PR. This would make the diff the same for all the core contracts so we wouldnt have to check them week to week anyways. I'm going to go ahead and revert back to the old code.

Reply:
> I think I agree with this. While it's cleaner to use the `ChainLog` in the tests, we might want the test to challenge our assumptions and configuration by explicitly testing addresses.
> 
> On the other hand, making spell construction easier is a huge part of why we have the `ChainLog`. It seems a shame not to lean on it here.
> 
> Perhaps a hybrid approach that gets us easy test construction, but also has a relatively static section that validates all `ChainLog` addresses? The same way we enumerate system and collateral config, we would have a test that makes sure `ChainLog` is correct.

I'm open to this idea, but we should make a separate ticket for it.

Reply:
I'd be in favor of keeping the hard-coded addresses accessible, one way to do this would be to add an external contract (like `test/rates.sol`) that has the addresses hardcoded, then we can just swap that contract out in spells-mainnet and spells-kovan and keep the rest of the logic the same.